### PR TITLE
echo: added -n, -e examples

### PR DESCRIPTION
**Pull request by AgamAgarwal**
Added examples for the options `-n` and `-e` for `echo`

### CODE REVIEW

File: pages/common/echo.md [hunk 1]
```diff
@@ -9,3 +9,11 @@
 - Print a message with environment variables:
 
 `echo {{"My path is $PATH"}}`
+
+- Print a message without the trailing newline:
+
+`echo -n {{"Hello World"}}`
+
+- Enable backslash escapes:
```
Comment (waldyrious) [inline pages/common/echo.md]: How about "Enable enhanced mode to include special characters (escaped with backslash)"? That way the -e maps to "enhanced"

Comment (AgamAgarwal) [inline pages/common/echo.md]: This sounds interesting.
Though, as per my knowledge, `-e` corresponds to "escapes".

Comment (waldyrious) [inline pages/common/echo.md]: You're right. In that case I'd suggest "Enable interpretation of backslash escapes (special characters)".

Comment (AgamAgarwal) [inline pages/common/echo.md]: Done

**Comment by ostera** [comment]
LGTM

**Pull request merged**